### PR TITLE
threads.dust integrated with html & css

### PR DESCRIPTION
**Pull request by janelleany**
bitmoji@kbooth1000 & @ponchieponcho please review. thanks.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -55,7 +55,6 @@ h1 {
 }
 
 .video-thumbnail-box {
-    background-image: url('file:///Users/jra/Desktop/DigitalCrafts/W9/Town-Square/images/video_thumbnail.jpg');
```
Comment (kbooth1000) [inline style.css:58]: this background-image url path can be from localhost:3000 instead of from 'file:///......'

**Comment by ponchieponcho** [comment]
Please remove .DS_Stores on a follow up commit :)

File: views/layout.dust [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
         </div>
+    </header>
+
+
+
+    <div>
+        {+body /}
     </div>
+
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (kbooth1000) [inline views/layout.dust]: We can get rid of this script link cause we're not using jquery anywhere.

File: style.css (hunk 1 above)
Comment (janelleany) [reply style.css:58]: Yep. In the final style.css script we use, that line of code will be deleted. Right now this line of code is pointing nowhere.

File: views/layout.dust (hunk 2 above)
Comment (janelleany) [reply views/layout.dust]: bitmoji

deleting

**Review by ponchieponcho — APPROVED** [review]
good enough

**Pull request merged**